### DISCUSSION TITLE
dashboard url with datetime range

### DISCUSSION THREAD

**Discussion by unihon** [apache/skywalking#11895] [Q&A] [answered]
![image](https://github.com/apache/skywalking/assets/42078581/40b9f938-24ff-4361-832a-4161001f6e0b)

Whether the URL supports carrying time interval information, the type is similar to grafana

**Comment by wu-sheng** [comment, marked as answer]
I think we don't support that. The metric widget has the permanent link generation mechanism, but not for the whole dashboard.

<img width="545" alt="image" src="https://github.com/apache/skywalking/assets/5441976/a939da59-b8f0-44c6-98a2-06ffdd6ec41a">
